2023-06-06 18:24:35,399 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Current SDK version is 0.15.3
2023-06-06 18:24:35,400 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Configure stats pid to 7728
2023-06-06 18:24:35,400 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Neko\.config\wandb\settings
2023-06-06 18:24:35,400 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Neko\OneDrive\Desktop\learning\Homework\mlops_zoomcamp\experiment-tracking\wnb\wandb\settings
2023-06-06 18:24:35,400 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-06 18:24:35,400 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-06 18:24:35,400 INFO    MainThread:7728 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'experiment-tracking\\wnb\\preprocess_data.py', 'program': 'C:\\Users\\Neko\\OneDrive\\Desktop\\learning\\Homework\\mlops_zoomcamp\\experiment-tracking\\wnb\\preprocess_data.py'}
2023-06-06 18:24:35,401 INFO    MainThread:7728 [wandb_init.py:_log_setup():507] Logging user logs to C:\Users\Neko\OneDrive\Desktop\learning\Homework\mlops_zoomcamp\experiment-tracking\wnb\wandb\run-20230606_182435-x2dlcf0c\logs\debug.log
2023-06-06 18:24:35,401 INFO    MainThread:7728 [wandb_init.py:_log_setup():508] Logging internal logs to C:\Users\Neko\OneDrive\Desktop\learning\Homework\mlops_zoomcamp\experiment-tracking\wnb\wandb\run-20230606_182435-x2dlcf0c\logs\debug-internal.log
2023-06-06 18:24:35,401 INFO    MainThread:7728 [wandb_init.py:init():547] calling init triggers
2023-06-06 18:24:35,401 INFO    MainThread:7728 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-06-06 18:24:35,401 INFO    MainThread:7728 [wandb_init.py:init():596] starting backend
2023-06-06 18:24:35,401 INFO    MainThread:7728 [wandb_init.py:init():600] setting up manager
2023-06-06 18:24:35,407 INFO    MainThread:7728 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-06-06 18:24:35,413 INFO    MainThread:7728 [wandb_init.py:init():606] backend started and connected
2023-06-06 18:24:35,417 INFO    MainThread:7728 [wandb_init.py:init():700] updated telemetry
2023-06-06 18:24:35,464 INFO    MainThread:7728 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-06-06 18:24:35,569 ERROR   MainThread:7728 [wandb_init.py:init():763] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-06-06 18:24:35,769 ERROR   MainThread:7728 [wandb_init.py:init():1168] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
Traceback (most recent call last):
  File "C:\Users\Neko\OneDrive\Desktop\learning\Homework\mlops_zoomcamp\venv\lib\site-packages\wandb\sdk\wandb_init.py", line 1150, in init
    run = wi.init()
  File "C:\Users\Neko\OneDrive\Desktop\learning\Homework\mlops_zoomcamp\venv\lib\site-packages\wandb\sdk\wandb_init.py", line 769, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-06-06 18:24:36,788 WARNING MsgRouterThr:7728 [router.py:message_loop():77] message_loop has been closed
